contracts/contracts/MACI.sol

```diff
@@ -188,15 +188,14 @@ contract MACI is IMACI, DomainObjs, Params, Utilities {
       revert InvalidPubKey();
     }
 
-    MaxValues memory maxValues = MaxValues({
-      maxMessages: uint256(MESSAGE_TREE_ARITY) ** _treeDepths.messageTreeDepth,
-      maxVoteOptions: uint256(MESSAGE_TREE_ARITY) ** _treeDepths.voteOptionTreeDepth
-    });
+    MaxValues memory maxValues = MaxValues({ maxMessages: 1000, maxVoteOptions: 25 });
```

Comment:
let's keep maxVoteOptions as `uint256(MESSAGE_TREE_ARITY) ** _treeDepths.voteOptionTreeDepth` and max messages as param by the poll deployer

Reply:
well actually we need a `VOTE_TREE_ARITY` constant in this contract and use it to calculate this `maxVoteOptions`

Reply:
also `maxMessages` should be a parameter not hardcoded

Reply:
Max messages parameter will be removed in the next milestone.

Reply:
ok but let's fix maxVoteOptions, it should not be hardcoded